packages/@cdk-cosmos/core/src/solar-system/solar-system-core-import.ts

```diff
@@ -32,6 +32,7 @@ export class SolarSystemCoreImport extends BaseConstruct implements ISolarSystem
       isolatedSubnetNames: ['App'],
       ...vpcProps,
     });
+    (this.vpc as any).zone = RemoteZone.import(this, 'VpcZone', this.singletonId('VpcZone'));
     this.zone = RemoteZone.import(this, 'Zone', this.singletonId('Zone'));
     this.privateZone = RemoteZone.import(this, 'PrivateZone', this.singletonId('PrivateZone'));
```

Comment:
If `privateZone` is not being exported, it won't error out while importing it right..?

Reply:
It wont, only if its trying to be consumed at cfn deployment